src/main/java/Duke.java

```diff
@@ -31,9 +34,40 @@ public static void main(String[] args) throws IOException  {
     }
 
     public static String promptEntry() {
-       return sc.next();
+        return sc.next();
     }
 
+    public static String getFormattedDate(String dateAndTime) {
```

Comment:
Good use of the SimpleDateFormat method from the Date Package, makes things a lot simpler